lib/TelephonyCredential.php

```diff
@@ -27,4 +27,41 @@ public static function classUrl()
         // Original function inside ApiResource.php
         return "/v2/telephony_credentials";
     }
+
+    /**
+     * Create an Access Token (JWT) for the credential.
+     *
+     * @param array|null $params
+     * @param array|string|null $options
+     *
+     * @return string
+     */
+    public function token($params = null, $options = null)
+    {
+        $url = $this->instanceUrl() . '/token';
+
+        $params = $params ?: [];
+        $headers = $options ?: [];
+
+        // This is basically the first half of Telnyx\ApiOperations\Request::_staticRequest() and we don't interpret as json unless there's an error
+        $opts = \Telnyx\Util\RequestOptions::parse($options);
+        $baseUrl = isset($opts->apiBase) ? $opts->apiBase : static::baseUrl();
+        $requestor = new \Telnyx\ApiRequestor($opts->apiKey, $baseUrl);
+
+        // Instead of calling the neatly packaged request() function, we're calling _requestRaw()
+        list($rbody, $rcode, $rheaders, $myApiKey) = $requestor->_requestRaw('post', $url, $params, $headers);
+
+        // Remove headers that don't need to be persistent across other requests
+        $opts->discardNonPersistentHeaders();
+
+        if ($rcode < 200 || $rcode >= 300) { // If there is an error then we need to interpret as a json and return the error object
+            $json = $requestor->_interpretResponse($rbody, $rcode, $rheaders);
```

Comment:
Should `_interpretResponse` be made public in [APIRequestor](https://github.com/team-telnyx/telnyx-php/pull/69/files#diff-4946db3aaa038c0ffbcbd3f8635f772c9c7163a2d407afbe3022742deca82d4eL380)

Reply:
_interpretResponse() is now public